### PR TITLE
chore(rpa): Add documentation for date-format property 

### PR DESCRIPTION
**Pull request by mboskamp**
Related to CAM-14199

### CODE REVIEW

File: content/user-guide/ext-client/_index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ menu:
 
 ---
 
-The **Camunda External Task Client** allows to set up remote Service Tasks for your workflow. There is a supported [Java](https://github.com/camunda/camunda-bpm-platform/clients/java)
+The **Camunda External Task Client** allows to set up remote Service Tasks for your workflow. There is a supported [Java](https://github.com/camunda/camunda-bpm-platform/tree/master/clients/java)
```
Comment (tmetzke) [inline content/user-guide/ext-client/_index.md:14]: Nice catch 👍

**Comment by mboskamp** [comment]
@christinaausley, would you mind giving this PR a quick proofread?

**Review by christinaausley — APPROVED** [review]
Minor editorial adjustments in latest commit. LGTM, but let me know if you have any questions.

**Comment by mboskamp** [comment]
Hey @christinaausley, thanks for going over the changes. One question: We use "Camunda External Task Client" and "Camunda RPA Bridge" to refer to product names (e.g. "Download the Camunda External Task Client."). Should we still lowercase them? I am fine with both ways. I just want to clarify for future occurrences. :)

**Comment by christinaausley** [comment]
@mboskamp Ah, I see. In this case, yes. The Camunda product should be capitalized. I'll add this to our glossary.

File: content/user-guide/ext-client/_index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ menu:
 
 ---
 
-The **Camunda External Task Client** allows to set up remote Service Tasks for your workflow. There is a supported [Java](https://github.com/camunda/camunda-bpm-platform/clients/java)
+The **Camunda external task client** allows you to set up remote service tasks for your workflow. There is a supported [Java](https://github.com/camunda/camunda-bpm-platform/tree/master/clients/java)
```
Comment (mboskamp) [inline content/user-guide/ext-client/_index.md:14]:
```suggestion
The **Camunda External Task Client** allows you to set up remote service tasks for your workflow. There is a supported [Java](https://github.com/camunda/camunda-bpm-platform/tree/master/clients/java)
```

**Comment by mboskamp** [comment]
@tmetzke, @christinaausley Thanks for your feedback :)

**Pull request merged**